### PR TITLE
Feature - move SQL dependency tracking to sep project

### LINKED ISSUE #131: Move SQL dependency tracking to separate library
**What feature do you want?**
Introduce a separate SQL package as part of v1.0 instead of relying on the SQL NuGet package now present in the Core library.

_Originally posted by @tomkerkhove in https://github.com/arcus-azure/arcus.observability/pull/130_

### PR DESCRIPTION
**Pull request by stijnmoreels**
Move the SQL package together with the related dependency tracking extensions to a seperate project.

Closes #131

### CODE REVIEW

File: docs/preview/features/writing-different-telemetry-types.md [hunk 1]
```diff
@@ -254,6 +264,8 @@ _logger.LogSqlDependency(connectionString, "my-table", "get-products", isSuccess
 // Output: "SQL Dependency sample-server for sample-database/my-table for operation get-products in 00:00:01.2396312 at 03/23/2020 09:32:02 +00:00 (Successful: True - Context: )"
 ```
 
+
+
 ### Measuring custom dependencies
```
Comment (tomkerkhove) [inline docs/preview/features/writing-different-telemetry-types.md:269]:
```suggestion
### Measuring custom dependencies
```

File: src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <AssemblyName>Arcus.Observability.Telemetry.Sql</AssemblyName>
+    <RootNamespace>Arcus.Observability.Telemetry.Sql</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Arcus.Observability.Telemetry.Core/Arcus.Observability.Telemetry.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Guard.Net" Version="1.2.0" />
```
Comment (tomkerkhove) [inline src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj:27]: Core already has this one so we can remove this one, no?

File: src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <PackageId>Arcus.Observability.Telemetry.Sql</PackageId>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <Description>Provides capability to improve Sql telemetry with Serilog in applications</Description>
```
Comment (tomkerkhove) [inline src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj:8]:
```suggestion
    <Description>Provides capability to improve SQL telemetry with Serilog in applications</Description>
```

**Review by tomkerkhove — APPROVED** [review]

File: src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj (hunk 2 above)
Comment (stijnmoreels) [reply src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj:27]: Yes, that's true, although I don't know if that's the best package management strategy. The IoT project also explicitly states this package so I wanted to be consistent here.

Comment (tomkerkhove) [reply src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj:27]: It's best to do that to keep the dependencies less spread

Comment (stijnmoreels) [reply src/Arcus.Observability.Telemetry.Sql/Arcus.Observability.Telemetry.Sql.csproj:27]: OK, let's do that!

**Pull request merged**